### DISCUSSION TITLE
有dns泄露的问题

### DISCUSSION THREAD

**Discussion by jj-a-li** [trojan-gfw/igniter#376] [Suspicious Problems]
连接后，本机dns和服务器dns同时可查询

**Comment by jj-a-li** [comment]
![IMG_20201127_071409](https://user-images.githubusercontent.com/72844255/100400938-f9d86900-3092-11eb-8049-6be4685abda5.jpg)

**Comment by oasiscifr** [comment]
Interesting issue, but more information is needed. What software is this? What specific problems does the result indicate?
@jj-a-li
